opm/autodiff/ParallelOverlappingILU0.hpp

```diff
@@ -110,30 +249,35 @@ class ParallelOverlappingILU0
       \param A The matrix to operate on.
       \param w The relaxation factor.
     */
-    ParallelOverlappingILU0 (const Matrix& A, const ParallelInfo& comm,
-                             field_type w)
-        : ilu_(A), comm_(comm), w_(w)
+    ParallelOverlappingILU0 (const Matrix& A, const int iluIteration, const field_type w )
```

Comment:
Of course, for better understanding we could rename iluiteration to n (like in the istl implementation).